website/app/styles/typography/mixins.scss

```diff
@@ -14,12 +14,10 @@
   @if ($family == "hashicorp-sans") {
     font-family: hashicorp-sans-variable, Geneva, Tahoma, Helvetica, Verdana, sans-serif;
     font-synthesis: none; // Safari makes this variable font too bold, compared to Chrome, and this fixes the issue
-  } @else if ($family == "sans") {
```

Comment:
[todo] If we decided to not use anymore the "metro web" custom font, we should remove it also from the `assets/fonts` folder, and from the `website/app/styles/typography/font-faces.scss` file.

Reply:
@heatherlarsen @KristinLBradley general comment: for me there was a specific reason to use a different font from the HDS body as body of the website: to visually differentiate what was an HDS component (inlined as image, or as code snippet preview) from what is actual body copy of the website.

In the same way that we don't use HDS component to build the HDS website, we should not use the HDS font (even if it's the system font) to build the HDS website.

But that's my opinion.

Reply:
Also, I _think_ that the custom font allowed a bit more granularity in terms of font-weight than the system font.

Reply:
Your reasoning above makes sense to me but will wait on hearing from @heatherlarsen and other @hashicorp/hds-design people on what they want to do. (If we want to keep this change then I'll go ahead and delete the metro web font file)

Reply:
I tend to agree that using a body font that is different from what is used in our components for differentiation purposes is logical. I took a brief skim through the deploy preview and have a couple of takeaways:

To me (and this may be because we spend a lot time looking at the site), there is a pretty noticeable difference between the two fonts, especially in the navigation (side bar specifically). The weights might need some additional adjustment, but they look very different.

**Before**
<img width="310" alt="Screenshot 2024-11-25 at 1 08 07 PM" src="https://github.com/user-attachments/assets/701cae81-f5b4-43ea-8fd4-163f89c888bb">

**After**
<img width="301" alt="Screenshot 2024-11-25 at 1 08 33 PM" src="https://github.com/user-attachments/assets/d04823d0-992b-4ea7-9998-b372c34ff466">

Additionally, the type-scale is different between what is on the website vs what is used in our components, I get a bit of a feeling like there's a mistake when components are are paired with body text. It's almost like they are intended to be the same, but are off by a small amount which stands out to me.

Again, might be due to visual familiarity with the website, and I'm not proposing a different typeface, just noting my first impression at the visual change.

Reply:
I agree with @jorytindall here. I think the difference is noticeable since we look at the site a lot but would still be enough for a casual user to notice something is "off"

I also agree with @didoo on wanting to differentiate the body content with components but I don't think this font is quite "different enough" to do that.